src/dynode.h

```diff
@@ -149,7 +149,8 @@ class CDynode : public dynode_info_t
     enum CollateralStatus {
         COLLATERAL_OK,
         COLLATERAL_UTXO_NOT_FOUND,
-        COLLATERAL_INVALID_AMOUNT
+        COLLATERAL_INVALID_AMOUNT,
+        COLLATERAL_INVALID_PUBKEY
```

Comment:
What is COLLATERAL_INVALID_PUBKEY used for.  I don't see it referenced.

Reply:
dynode.cpp and governance-object.cpp

Reply:
https://github.com/dashpay/dash/pull/1783/files